gammapy/maps/tests/test_geom.py

```diff
@@ -266,3 +266,81 @@ def test_squash():
     assert_allclose(axis.edges[0], ax_sq.edges[0])
     assert_allclose(axis.edges[-1], ax_sq.edges[1])
     assert_allclose(ax_sq.center, 1.5)
+
+
+@pytest.fixture(scope="session")
+def energy_axis_ref():
+    edges = np.arange(1, 11) * u.TeV
+    return MapAxis.from_edges(edges, name="energy")
+
+
+
+class TestMapAxisGropuping:
```

Comment:
TestMapAxisGrouping:

Reply:
If you have only static methods, why do you need to make this a class?

Reply:
Good question! I think I just prefer to group related tests, but as there is no `MapAxisGrouping` class, there is no real point. I will change it...